testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -159,17 +160,19 @@ TestServer_clientID_booted : UnitTest {
 
 		remote1.remove;
 		thisProcess.platform.killProcessByID(serverPid);
+		exitCond.waitFor(5); // wait for the server process to exit
```

Comment:
Sorry I could have been more thorough in my previous comment.

If this does time out, it could be good to catch it. I did this in `TestCoreUGens`:
https://github.com/supercollider/supercollider/blob/7b97f535befdd4899bb1f2f4b9c6fd92b84a7870/testsuite/classlibrary/TestCoreUGens.sc#L342-L343

But there could be a more proper pattern to use.

Reply:
Got it! How's it now?

Reply:
Looks good. One detail I included in the timeout message in my example above was the name of the failing test. 

This may not actually be needed if `assert` also posts the method that called it when it throws... are you able to test that (like with a 0 or ver short timeout)?

Reply:
Yes, it does post, that's why I omitted it: `FAIL: a TestServer_clientID_booted: test_repeatedRemoteLogin - TIMEOUT: server process did not quit`